owner/manager_test.go

```diff
@@ -11,12 +11,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build !windows
```

Comment:
Maybe the original is better: if someone adds another test that is not related to `NewClusterV3`, it is ignored. It is not likely tests in the packages will change, though.

Reply:
Updated.